test/test_playbooks/domain.yml

```diff
@@ -0,0 +1,47 @@
+---
+- hosts: fixtures
+  gather_facts: false
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - name: Load domain test-config
+    include_vars:
+      file: domain_vars.yml
+  - include_tasks: tasks/location.yml
+    vars:
+      location_name: "{{ item }}"
+      location_state: "present"
+    with_items: "{{ domain_locations }}"
+  - include_tasks: tasks/organization.yml
+    vars:
+      organization_name: "{{ item }}"
+      organization_state: "present"
+    with_items: "{{ domain_organizations }}"
```

Comment:
Can you clean up the entities again afterwards?

Reply:
I will have to think/learn about how to save the _change_-state in a variable to make sure that only entities that have not existed before are deleted

Reply:
That would even raise the standards for our test_playbooks. ;)

Reply:
Went the easy way by just doing a delete (for now :-) )